setup.py

```diff
@@ -36,17 +36,18 @@
 setup(
     name='rev_ai',
     version=version,
-    description="Rev.ai makes speech applications easy to build!",
+    description='Rev.ai makes speech applications easy to build!',
```

Comment:
Why is it necessary to convert all quotes to single quotes?

If you are establishing a coding style, I'm curious where it is documented. Speech team's coding style is outlined in https://speech-knowledge.rev.com/intro/coding-conventions (`flake8 --ignore=E122,E126,E127,E501`)

Reply:
Thought it was better to be consistent, and my preference was towards single quotes. As for documentation of coding styles - we don't have documentation of Python's but probably a good idea - will most likely be on `dev-wiki` if we do create one